packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,41 @@
+# Dropdown
+
+This is Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```html
+<Dropdown>Dropdown</Dropdown>
+```
+
+Advanced example usage:
+
+```html
+<Dropdown elementType="span" isOpen>Dropdown content</Dropdown>
+```
+
+Without lexer:
+
+```twig
+{% embed "@spirit/dropdown.twig" with { props: {
+    isOpen: true
+    elementType: 'span'
+}} %}
+    {% block content %}
+          Dropdown content
+    {% endblock %}
+{% endembed %}
+```
+
+## API
+
+| Prop name     | Type     | Default | Required | Description                  |
+| ------------- | -------- | ------- | -------- | ---------------------------- |
+| `class`       | `string` | `null`  | no       | Custom CSS class             |
```

Comment:
To be consitent with the implementation of `Collapse`, there should be also the `id` and `breakpoint` options.

Reply:
`id` is added via `mainProps`.

Reply:
OK, I think we should get back to it later to make it uniform across components.